Dockerfile

```diff
@@ -61,7 +61,10 @@ RUN apt update -y -q && apt upgrade -y -q && apt update -y -q && \
     re2c \
     perl \
     cpanminus \
-    openssh-client
+    openssh-client \ 
+    ocaml-nox \
+    libgmp-dev \
+    opam
```

Comment:
could you add a comment saying these are needed for CompCert? We might split the docker, so it will help us in the future.